src/libc/pthread/thread.rs

```diff
@@ -173,11 +174,18 @@ fn pthread_mach_thread_np(env: &mut Environment, thread: pthread_t) -> mach_port
     host_object.thread_id.try_into().unwrap()
 }
 
+// POSIX usleep
+fn usleep(_env: &mut Environment, useconds: u32) {
+    let duration = Duration::from_millis(u64::from(useconds) / 1000);
```

Comment:
Rust offers a `Duration::from_micros` method so there's no need for a conversion here.